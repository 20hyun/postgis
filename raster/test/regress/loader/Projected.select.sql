SELECT st_srid(rast) from loadedrast limit 1;
SELECT st_extent(ST_SnapToGrid(rast::geometry,1)) from loadedrast;
