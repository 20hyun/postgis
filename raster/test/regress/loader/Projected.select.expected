3857
-20000000
3857
-20000000
